tests/testthat/test_oligoNucleotideEnrichment.R

```diff
@@ -0,0 +1,20 @@
+test_that("oligoNucleotideEnrichment works not correct", {
+library(BSgenome.Hsapiens.UCSC.hg19)
+filepath =system.file("extdata", "examplePattern.fa", package="ChIPpeakAnno")
+peaks = GRanges(seqnames=c("chr17","chr3"),
+                 IRanges(start=c(41275784,10076141),
+                         end=c(41276382,10076732),
+                         names=paste0("peak",1:2)))
+
+result <- oligoNucleotideEnrichment(filepath=filepath,
+peaks=peaks,
+genome=Hsapiens,
+methodBackground="selectChromRandomly",
+times=20,
+alpha=0.1)
+
+
+expect_equal(result[1:2,3], c(0.0009829383,0.0033579623))
```

Comment:
Will this too stringent? I am not sure different OS will export exactly same values. try to use tolerance parameter or use a simulation data for test.

Reply:
added tolerance parameter in this case.